cg/meta/workflow/analysis.py

```diff
@@ -209,14 +209,14 @@ def add_pending_trailblazer_analysis(self, case_id: str) -> None:
         self.check_analysis_ongoing(case_id)
         self.trailblazer_api.add_pending_analysis(
             case_id=case_id,
-            email=environ_email(),
             analysis_type=self.get_application_type(
                 self.status_db.get_case_by_internal_id(case_id).links[0].sample
             ),
-            out_dir=self.get_job_ids_path(case_id).parent.as_posix(),
             config_path=self.get_job_ids_path(case_id).as_posix(),
+            out_dir=self.get_job_ids_path(case_id).parent.as_posix(),
             slurm_quality_of_service=self.get_slurm_qos_for_case(case_id),
-            data_analysis=str(self.workflow),
+            email=environ_email(),
+            workflow=self.workflow,
```

Comment:
Will this be serialized in a json? Thinking if we might have a serialization issue with the subtle difference between str(enum) and enum.

Reply:
Seem to work like this in testing. Let's deploy but be aware of this if issues arise.